packages/client/src/components/app/Button.svelte

```diff
@@ -58,15 +59,7 @@
     class:active
   >
     {#if icon}
-      <svg
-        class:hasText={componentText?.length > 0}
-        class="spectrum-Icon spectrum-Icon--size{size.toUpperCase()}"
-        focusable="false"
-        aria-hidden="true"
-        aria-label={icon}
-      >
-        <use xlink:href="#spectrum-icon-18-{icon}" />
-      </svg>
+      <i use:styleable={$component.styles} class="{icon} {size}" />
```

Comment:
We should remove the `use:styleable` here - that's specifically for the button itself rather than the icon, so could have unintended side effects if also applied here.

Reply:
Removed 👍